src/azure-cli/azure/cli/command_modules/storage/operations/azcopy.py

```diff
@@ -81,10 +81,14 @@ def storage_fs_directory_copy(cmd, source, destination, recursive=None, **kwargs
     azcopy.copy(source, destination, flags=flags)
 
 
-def storage_blob_sync(cmd, client, source, destination, exclude_pattern=None, include_pattern=None,
-                      exclude_path=None):
+def storage_blob_sync(cmd, client, source, destination, delete_destination=None, exclude_pattern=None,
```

Comment:
```suggestion
def storage_blob_sync(cmd, client, source, destination, delete_destination=True, exclude_pattern=None,
```
In this way, `default=True` will automatically show in help message